bin/make/test_unit.mk

```diff
@@ -0,0 +1,5 @@
+
+test_unit:
+	@echo
+	@echo '+++ run unit tests +++'
+	@vendor/bin/simple-phpunit --testsuite "unit tests"
```

Comment:
To be consistent, why don't we call this suite only "unit"? ("phpcr", "orm", "unit")

Reply:
we do not have a difference of persistence in unit tests until now. but we do have integration/functional tests which are different on persistence layer.

Reply:
i think wouter meant that its odd that the other suites are called "phpcr" and "orm" while this one is called "unit tests". we could be consistent and call them "phpcr", "orm" and "unit". or maybe "functional tests with phpcr", "functional tests with orm" and "unit tests"?

Reply:
ah now i got it. I think the last sounds better, but i would rename the the files and tasks then too.